webauthn-rs-core/src/crypto.rs

```diff
@@ -143,9 +144,13 @@ where
     }
 }
 
+/// TpmSanData
```

Comment:
I feel like this is to silence a warning for undocumented public features, but this is already public, and this comment doesn't really explain what this structure is.